base/Dockerfile

```diff
@@ -100,6 +100,10 @@ RUN apk add --no-cache \
     && apk del .build-dependencies \
     && rm -rf /usr/src/arp-scan
 
+# PicoTTS
+RUN apk add --no-cache \
+        picotts --repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing

```

Comment:
We should not use edge. As pico depends on musl, and the musl lib currently on edge isn't compatible with the stable version; this will break.

Instead, if we want to include this, we need to build it from source.

Reply:
Normally, I wouldn't even bother you guys to include something from "edge."

But from layman's eye,  it seems that the picotts code is relatively stable and hasn't been modified for ages.

I did a basic sanity test on my own and made sure the picotts actually worked before I did the pull request.  Nothing broke based upon my very limited test (docker on x86_64 linux host).

On the other hand,  since I am absolutely clueless about intricacies of various libc libraries, I am going to assume your concern is real.  

My question to you is,  what is the best way to moving this forward?  compile picotts against a stable musl?  switch to glibc altogether?

Reply:
There is no glibc here, as this is all Alpine Linux and thus musl.

The concern is real and actually a current issue. If Pico is added to this base image, it needs to be compiled from the source in-place, right here in this Dockerfile.

Reply:
So, if I see it correctly, this would result into a multi-image build: The first image is to create picotts from source using regulare build utilities. (automake, make, gcc). Since these tools are not required in production, the line would result into a `COPY --from ... ` expression, if I am right. This would definitely be a harder worker to do. 

@frenck Do you have experiences with that or should I try to get a working copy for it next week during my vacation?

Reply:
not realy, just a build like we do for other software... Keep it simple make it smart.

Reply:
Yeah, I just saw that build-utils are installed in the base image. I usually skip this package, if it is not required at runtime.

Reply:
Ok, I found it:

docker run -it alpine:3.12

$ apk update && apk add automake autoconf git libtool popt-dev git build-base && mkdir -p /tmp/pico && https://github.com/naggety/picotts.git /tmp/pico && cd /tmp/pico/pico && ./autogen.sh && ./configure && make && make install && cd - && rm -rfv /tmp/pico

This should build pico-tts

cc @kngharv